mailet/standard/src/main/java/org/apache/james/transport/mailets/AmqpForwardAttribute.java

```diff
@@ -106,32 +110,40 @@ public void init() throws MailetException {
 
     @Override
     public void service(Mail mail) throws MailetException {
-        if (mail.getAttribute(attribute) == null) {
-            return;
-        }
-        Stream<byte[]> content = getAttributeContent(mail);
-        try {
-            sendContent(content);
-        } catch (IOException e) {
-            LOGGER.error("IOException while writing to AMQP: {}", e.getMessage(), e);
-        } catch (TimeoutException e) {
-            LOGGER.error("TimeoutException while writing to AMQP: {}", e.getMessage(), e);
-        } catch (AlreadyClosedException e) {
-            LOGGER.error("AlreadyClosedException while writing to AMQP: {}", e.getMessage(), e);
-        }
+        getAttributeContent(mail)
+            .ifPresent(content -> {
+                try {
+                    sendContent(content);
+                } catch (IOException e) {
```

Comment:
I think we can group those exceptions, as the exception root cause will be in the stacktrace & they are all handled the same way.